dist/api_data.py

```diff
@@ -983,6 +983,12 @@ def __ge__(self, other):
         size and the default config would extend log files in allocations of
         the maximum log file size.''',
         type='list', choices=['data', 'log']),
+    Config('file_handle_close_sync', 'true', r'''
+        flush modified files to stable storage independent of checkpoints when
+        closing existing file handles and opening an exclusive file handle. If
+        set to false, certain API calls that require exclusive access to files may
+        return EBUSY to caller''',
```

Comment:
Further to Sue's comment. We generally talk about tables at the API level instead of files. "control whether to flush modified tables to table storage independent of a global checkpoint when acquiring exclusive access to a table. If set to false, API calls that require exclusive access to tables will return EBUSY if there have been changes made to the table since the last global checkpoint."

Reply:
Fixed.